src/Utils.cc

```diff
@@ -167,18 +167,11 @@ void enforceConfigurablePolicyCondition(
 }
 
 /////////////////////////////////////////////////
-void throwOrPrintErrors(const sdf::Errors& _errors)
+void throwOrPrintErrors(sdf::Errors& _errors)
```

Comment:
Shouldn't have to make this change if the member function is `const`.

Reply:
Changed back in https://github.com/gazebosim/sdformat/pull/1220/commits/6483e3ab20de09fc28f83e16ed45a67297419466.